applications/rag/README.md

```diff
@@ -165,7 +166,7 @@ This step generates the vector embeddings for your input dataset. Currently, the
 
 ### Launch the Frontend Chat Interface
 
-1. Setup port forwarding for the frontend [TBD: Replace with IAP]: `kubectl port-forward service/rag-frontend -n ${NAMESPACE:?} 8080:8080 &`
+1. Access from the frontend domain.
```

Comment:
Seems to go from 1 to 9. Also, needs more clarity on the instruction.

Reply:
Markdown will auto fix that. But for sure I can fix it.